match/lib/match/runner.rb

```diff
@@ -171,6 +172,8 @@ def fetch_provisioning_profile(params: nil, certificate_id: nil, app_identifier:
       installed_profile = FastlaneCore::ProvisioningProfile.install(profile, keychain_path)
       parsed = FastlaneCore::ProvisioningProfile.parse(profile, keychain_path)
       uuid = parsed["UUID"]
+      Fastlane::Actions.lane_context[:MATCH_PROFILES_UUID] = Fastlane::Actions.lane_context[:MATCH_PROFILES_UUID].to_a << uuid
+      Fastlane::Actions.lane_context[:MATCH_PROFILES_NAME] = Fastlane::Actions.lane_context[:MATCH_PROFILES_NAME].to_a << parsed["Name"]
```

Comment:
`Fastlane::Actions.lane_context` shouldn't get set in the `match/running.rb`. We should find a way to bring these values outside of runner so they can be set in the match action instead. I would start with looking how other actions/tools do this 😊 This should get rid of your rubocop error as well.

Reply:
@joshdholtz I think i got it, i pushed changes. Thanks a lot 🙏